etl/data_enhancement/enhancement_duplicates/JsonPost.py

```diff
@@ -1,9 +1,12 @@
 from copy import deepcopy
+from shared.LoggerFactory import LoggerFactory
 
 
 class JsonPost:
     """Handles the data to find duplicates."""
 
+    logger = LoggerFactory.get_enhancement_logger()
```

Comment:
There should be a consistent way of using the logger instance. In the Enhancer class you defined it as an class variable and now you are defining it as an static variable. Both should be used the same way (If the logger is used inside of a class).

Reply:
I disagree. The class only contains static methods and therefore it is not possible to interact with a instance attribute like with the other logger calls. the method could be converted to a non static method, but eventually there will be cases where a static method is needed and thus eventually logging will either have to be done using a class variable, or by using one of the 'get_logger' methods (like 'LoggerFactory.get_enhancement_logger().debug(message)). This was in my opinion the preferable option which is not too different from the other uses. in other places the static methods have only been changed because a logger that is connected to a specific instant of a child class was needed, not to maintain consistency in the logging calls. in other places logging is done via lokal variables so the logging calls are already inconsistent. technically they are consistent in the way that you access the loggerinstance in some way and use the logger. if the access happens via self.logger or JsonPost.logger is a negligible difference in my opinion